cores/cv32e40p/user_manual/source/apu.rst

```diff
@@ -7,6 +7,9 @@ Auxiliary Processing Unit Interface
 -----------------------------------
 
 :numref:`Auxiliary Processing Unit interface signals` describes the signals of the Auxiliary Processing Unit interface.
+This interface is used to send decoded instructions to an external co-processor that receives the operands and the operation and returns the result of such operation. The CV32E40P core uses it to process RV32F instructions (see Chapter 11 of the `RISC-V Instruction Set Manual, Volume I: User-Level ISA`).
```

Comment:
Are _all_ decoded instructions forwarded to the apu_*_o interface, or just instructions not handled by the core?  Is the APU required to process forwarded instructions?   What happens if it does not?